sdks/python/pom.xml

```diff
@@ -226,8 +226,8 @@
                 <configuration>
                   <executable>${python.user.base}/bin/tox</executable>
                   <arguments>
-                    <argument>-e</argument>
-                    <argument>ALL</argument>
+                    <!--Testenv that are only defined in envlist will be triggered.-->
+                    <argument>envlist</argument>
```

Comment:
Where do we set the value of `envlist`?

Reply:
cover will be triggered if running `tox -e ALL`. Since I didn't add cover to envlist in tox.ini, Precommit will not run cover in this case. 

This approach has two cons:
1. cover is not belong to default tox testenv.
2. testenv changes will not reflect in Precommit if they are not defined in envlist.

I tried to find a tox command without envlist to exclude cover in execution but failed.

Reply:
How long it takes to run coverage jobs?

Could you add a comment here and `tox.ini` that new environments will be excluded by default unless explicitly added to `envlist`.

Reply:
will do.

according to [this Postcommit run](https://builds.apache.org/job/beam_PostCommit_Python_Verify/4008/), `cover` ran 1488 tests in 245.447s and `py27gcp` ran same test set in 158.642s.

Reply:
OK. Let's keep it separate.